pkg/alertmanager/multitenant.go

```
@@ -198,13 +193,18 @@ func NewMultitenantAlertmanager(cfg *MultitenantAlertmanagerConfig, cfgCfg confi
 		go peer.Settle(context.Background(), cluster.DefaultGossipInterval)
```

I can't comment on non changed lines, but I've noticed above we do `os.Exit(1)`. I don't think it's correct. We should return the the error.

---

I agree, not something I noticed in the past but I can take care of this in a separate PR

---

I opened #2047 to fix this.